src/components/Scene.jsx

```diff
@@ -14,6 +14,7 @@ export default function Scene({sceneModel}) {
     traitsLeftEye,
     traitsRightEye,
     setControls,
+    setCamerDegrees,
```

Comment:
setCamerDegrees should be setCameraDegrees

Reply:
[fix typo](https://github.com/webaverse-studios/CharacterCreator/pull/152/commits/f9aade3984770580aa3cd608f34398e9af9be061)